### PR TITLE
Create Dot Props

### PR DESCRIPTION
**Pull request by jihu02**
## Description

Create Badge Dot Props

## Related Issues

-

## Tests

I added the following tests:

-

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Badge/index.tsx [hunk 1]
```diff
@@ -24,8 +25,8 @@ const StyledView = styled.View`
   align-items: center;
   opacity: ${(props: badgeProps): number =>
     props.count === 0 ||
-    props.count! <= props.maximumValue! ||
-    !props.opacityVisible
+      props.count! <= props.maximumValue! ||
+      !props.opacityVisible
       ? 1
       : 0.6};
```
Comment (JeffGuKang) [inline main/Badge/index.tsx:31]: I think it's better to move this part inside the component because the logic is complex.

**Review by JeffGuKang — COMMENTED** [review]
The first letter of the interface should always be capitalized. 
So `badgeProps` has to be changed.

File: main/Badge/index.tsx (hunk 1 above)
Comment (meohyun2) [reply main/Badge/index.tsx:31]: we think this logic will filter 2 conditions,
1. count is more than max count 
2. when count is more than max count, User`s option(Opacity Visible) will defined component`s opacity
but i dont know how to move this logic inside the component, not style code..

Comment (JeffGuKang) [reply main/Badge/index.tsx:31]: It would be more readable to use conditional statements before `render`.

**Pull request merged**